compose.go

```diff
@@ -402,3 +416,29 @@ func which(binary string) error {
 
 	return err
 }
+
+func readSTDIN() ([]byte, error) {
+	return ioutil.ReadAll(os.Stdin)
+}
+
+func (dc *LocalDockerCompose) makeTmpFile() error {
+	data, err := readSTDIN()
+	if err != nil {
+		return err
+	}
+
+	file, err := ioutil.TempFile("", "__tmp_")
```

Comment:
Do you foresee any collision between multiple runs? Do you think we could use the compose identifier as part of the file name in that case 🤔?

Reply:
fixed it